tests/run-make/broken-pipe-no-ice/rmake.rs

```diff
@@ -11,8 +11,6 @@
 // Internal Compiler Error strangely, but it doesn't even go through normal diagnostic infra. Very
 // strange.
 
-#![feature(anonymous_pipe)]
-
```

Comment:
Can you avoid modifying this test in *this* PR? It shouldn't be using unstable features, and we're trying to fix `rmake.rs` to be compiled with stage 0 (possibly beta) rustc (and fix this test) in #137373 and #137537.

Reply:
I think it would break it since the feature is now stablised?

the additional `#[feature(anonymous_pipe)]` would error

Reply:
It's currently doing something funny like building `rmake.rs` with a stage 1 compiler. Let me try to run this locally.